frontends/main/src/page-components/SearchDisplay/ResourceCategoryTabs.tsx

```diff
@@ -62,7 +62,11 @@ const ResourceCategoryTabContext: React.FC<{
   activeTabName: string
   children: React.ReactNode
 }> = ({ activeTabName, children }) => {
-  return <TabContext value={activeTabName}>{children}</TabContext>
+  return (
+    <TabContext value={activeTabName}>
+      <>{children}</>
```

Comment:
yes - this resolved an issue with some hydration error which to my knowledge was happening because next js has some caveats around what tags can be top-level. this appeared to resolve it on my end